crates/pathfinder/src/state/sync.rs

```diff
@@ -330,24 +328,26 @@ async fn update_sync_status_latest(
     use crate::rpc::types::{BlockNumberOrTag, Tag};
     let poll_interval = head_poll_interval(chain);
 
+    let starting = NumberedBlock::from((starting_block_hash, starting_block_num));
+
     loop {
         match sequencer
             .block_by_number(BlockNumberOrTag::Tag(Tag::Latest))
             .await
         {
             Ok(block) => {
-                let latest_hash = block.block_hash.unwrap();
-                let latest_num = block.block_number.unwrap();
+                let latest = {
+                    let latest_hash = block.block_hash.unwrap();
+                    let latest_num = block.block_number.unwrap();
```

Comment:
Wanted to keep these on separate lines for the unwrap message, should that ever become reported.